plugins/Monolog/config/config.php

```diff
@@ -17,6 +17,8 @@
         'file'     => 'Piwik\Plugins\Monolog\Handler\FileHandler',
         'screen'   => 'Piwik\Plugins\Monolog\Handler\WebNotificationHandler',
         'database' => 'Piwik\Plugins\Monolog\Handler\DatabaseHandler',
+        'syslog' => 'Piwik\Plugins\Monolog\Handler\SyslogHandler',
+        'errorlog' => 'Piwik\Plugins\Monolog\Handler\ErrorLogHandler',
```

Comment:
These classes don't exist, they need to match the actual monolog classes, for example, `\Monolog\Handler\ErrorLogHandler::class`

Reply:
Same applies for the references below.